webclient/xml_generator/models.py

```diff
@@ -44,7 +44,23 @@ def check_if_task_queued(class_name, pk, task_name):
     return False
 
 
-class ModelWithMetadata(models.Model):
```

Comment:
@jnihnat Můžu tady jen poprosit o vysvětlení, proč se toto definuje tady a ne někde v core? Toto se týká XML generátoru, jak to spolu souvisí?

Reply:
V core models to byt nemuze lebo by to spusobilo cyklicky import, proto je to na stejnem miste jak metadata model.